Tests/AppTests/Controllers/User/PushNotificationControllerTest.swift

```diff
@@ -0,0 +1,123 @@
+import XCTest
+import Testing
+import HTTP
+import Sockets
+import Fluent
+@testable import Vapor
+@testable import App
+
+// swiftlint:disable superfluous_disable_command
+// swiftlint:disable force_try
+class PushNotificationControllerTest: TestCase {
+
+  override func setUp() {
+    super.setUp()
+    do {
+      try drop.truncateTables()
+    } catch {
+      XCTFail("Droplet set raise exception: \(error.localizedDescription)")
+      return
+    }
+  }
+
+  func testThatPushTokenIsRegistered() throws {
+
+    let user = User()
+    try! user.save()
+
+    let pushToken = String.randomValue
+
+    let body = try! JSON(node: [ "push_token": pushToken ])
```

Comment:
removed space after leading and  before trailing square bracket

Reply:
))

Reply:
))) fixed